docs/migrating.md

```diff
@@ -1,3 +1,28 @@
-## Migrating from AppOptics to SolarWinds APM
 
-WIP
\ No newline at end of file
+### Table of Contents
+[Migrating AppOptics => SolarWinds APM](#aotosw)<br>
+
+<a name="aotosw"></a>
+
+## appoptics-apm migration guide to solarwinds-apm
+
+If you were using `appoptics-apm` out-of-the-box with **no** custom instrumentation and are migrating to a similar setup with `solarwinds-apm`, then follow the installation instructions for `solarwinds-apm`. Instrumentation should work as before (reporting to SolarWinds rather than AppOptics). You can stop reading now. Following applies only to migrating custom instrumentation.
+
+### Background
+
+There are two main changes to the API:
+
+- Agent now support W3C Trace Context propagation. Due to the implementation chosen, some functions related to trace now expect a traceparnet and tracestate duo rather than a single xtrace value. Note that "tracestate" expected is in the format of spanId only.
```

Comment:
Typo: traceparent, instead of traceparnet

Also a question about `tracestate` propagation in the Node agent! In NH Python, tracestate is a string of comma-separated KV pairs, e.g. `"tracestate": "sw=1111222233334444-01,some-other-kv=abcdef"`. This is not the case for NH Node/JS, right? e.g. `"tracestate": 1111222233334444". Just wondering, seems nice 😉

Reply:
fixed.

and...

*"tracestate"*, the word, in the W3C Trace Context, refers to a string of comma delimited key value pairs. It is meant to maintain state (dah) over stateless http requests (dha). It's a cookie... 🍪 

While the trace state meant to be processed within the context of the http request, it was decided to break the probe, agent and bindings api barriers and send the value down to oboe for "decisions". This design choice is debatable, and was debated, however, by now, this is "spilled milk under the bridge". 

The value oboe uses however is not "tracestate" in its "raw" form but rather the the value extracted from *our* key-value pair. oboe refers to that value as ["tracestate"](https://github.com/librato/solarwinds-apm-liboboe/blob/165c750abc29a883ec744c5fec1035cbe8522454/liboboe/oboe.h#L237), and so do the bindings and this bubbles up to the api.

Reply:
Ahhhh ok right! Thanks Ron 🍪 🥛